src/lib/vocab/space/spaceServices.ts

```diff
@@ -52,13 +51,29 @@ export const UpdateSpaceService: ServiceByName['UpdateSpace'] = {
 	action: UpdateSpace,
 	transaction: true,
 	perform: async ({repos, params}) => {
+		log.debug('[updateSpace] updating space');
 		const {space_id, actor, ...partial} = params;
 		const space = await repos.space.findById(space_id);
 		if (!space) {
 			return {ok: false, status: 404, message: 'no space found'};
 		}
 
 		await checkPolicy('update_space', actor, space.hub_id, repos);
+
+		if (HOME_VIEWS.includes(space.view)) {
+			throw new ApiError(405, 'Home spaces cannot be updated');
```

Comment:
following the same pattern as below with deleted and in other services:

```suggestion
			throw new ApiError(405, 'cannot update home space');
```